cmd/bootstrap.go

```diff
@@ -43,11 +43,13 @@ func bootstrap() error {
 	}
 
 	themeName := getThemeName()
-	kit.Printf(
-		"Attempting to create theme %s from %s",
-		kit.YellowText(themeName),
-		kit.YellowText(zipLocation),
-	)
+	if verbose {
+		kit.Printf(
```

Comment:
I don't really want to do that because it there might be instances where I want it to output a message even when verbose is set to false. The other way would be to have logging levels but I don't want to overcomplicate things. If it starts to get too out of hand then I will approach it again.